### PR TITLE
provide error log by default

### LINKED ISSUE #659: [CSI] lack of info when given wrong conf file
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
/kind bug
> /kind feature

**What happened**:
start  ./cinder-csi-plugin  by accidently give a wrong cloud conf file , just give following info and the process stopped without helpful info
```
[root@jichen-openstack-1 cloud-provider-openstack]# ./cinder-csi-plugin --endpoint tcp://127.0.0.1:10000 --cloud-config acloud.conf --nodeid CSINodeID --cluster ClusterID
I0626 06:46:15.032438   11171 driver.go:57] Driver: cinder.csi.openstack.org version: 1.0.0
I0626 06:46:15.032511   11171 driver.go:88] Enabling controller service capability: LIST_VOLUMES
I0626 06:46:15.032517   11171 driver.go:88] Enabling controller service capability: CREATE_DELETE_VOLUME
I0626 06:46:15.032525   11171 driver.go:88] Enabling controller service capability: PUBLISH_UNPUBLISH_VOLUME
I0626 06:46:15.032529   11171 driver.go:88] Enabling controller service capability: CREATE_DELETE_SNAPSHOT
I0626 06:46:15.032536   11171 driver.go:88] Enabling controller service capability: LIST_SNAPSHOTS
I0626 06:46:15.032542   11171 driver.go:100] Enabling volume access mode: SINGLE_NODE_WRITER
I0626 06:46:15.032551   11171 driver.go:110] Enabling node service capability: STAGE_UNSTAGE_VOLUME
```
**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- openstack-cloud-controller-manager version:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by jichenjc**
given error info by default to make admin/user easier to find
where's wrong settings

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
fixes #659 
**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by adisky** [comment]
@jichenjc you need to rebase your PR, to avoid v1.12 failure(It will not run)

File: cmd/cinder-csi-plugin/main.go [hunk 1]
```diff
@@ -101,21 +101,21 @@ func handle() {
 	//Intiliaze mount
 	mount, err := mount.GetMountProvider()
 	if err != nil {
-		klog.V(3).Infof("Failed to GetMountProvider: %v", err)
+		klog.Infof("Failed to GetMountProvider: %v", err)
```
Comment (lingxiankong) [inline cmd/cinder-csi-plugin/main.go]: All those logs will be printed when there is some error occurs, is it necessary to change to warning log level if the particular error doesn't break the process?

Comment (adisky) [reply cmd/cinder-csi-plugin/main.go]: agreed, here it is not required, it is sufficient to do it here only
https://github.com/kubernetes/cloud-provider-openstack/blob/master/cmd/cinder-csi-plugin/main.go#L120

Comment (jichenjc) [reply cmd/cinder-csi-plugin/main.go]: ok

File: pkg/csi/cinder/openstack/openstack.go [hunk 2]
```diff
@@ -128,14 +128,14 @@ func GetConfigFromFile(configFilePath string) (Config, gophercloud.EndpointOpts,
 	var cfg Config
 	config, err := os.Open(configFilePath)
 	if err != nil {
-		klog.V(3).Infof("Failed to open OpenStack configuration file: %v", err)
+		klog.Infof("Failed to open OpenStack configuration file: %v", err)
```
Comment (adisky) [inline pkg/csi/cinder/openstack/openstack.go]: let them be at log level 3

**Comment by hogepodge** [comment]
@adisky this is such a simple patch, are you ok with us overriding the failed checks?

**Comment by adisky** [comment]
@hogepodge yes, lets merge this

**Pull request merged**